DESCRIPTION

```diff
@@ -1,15 +1,46 @@
 Package: sageseqr
-Title: What the Package Does (One Line, Title Case)
+Title: Identify differentially expressed genes counts data.
 Version: 0.0.0.9000
 Authors@R: 
-    person(given = "First",
-           family = "Last",
+    person(given = "Kelsey",
+           family = "Montgomery",
            role = c("aut", "cre"),
-           email = "first.last@example.com",
-           comment = c(ORCID = "YOUR-ORCID-ID"))
-Description: What the package does (one paragraph).
-License: What license it uses
+           email = "kelsey.montgomery@sagebase.org",
+           comment = c(ORCID = "0000-0002-0126-8944"))
+Description: This workflow normalizes counts, performs model selection from known 
+    covariates using multivariate regression, automates outlier analysis using a 
+    customizable threshold and performs differential expression analysis for repeated
+    measures.
+License: MIT + file LICENSE
 Encoding: UTF-8
 LazyData: true
 Suggests: 
     testthat (>= 2.1.0)
+RoxygenNote: 7.1.0
+biocViews:
+Imports:
+    biomaRt,
+    config,
```

Comment:
```suggestion
    config,
    CovariateAnalysis,
```

The DESCRIPTION still needs to have CovariateAnalysis under Imports in addition to having 

```
Remotes:
    th1vairam/CovariateAnalysis@dev
```

Imports tells R which packages are required, and Remotes tells it where to find them if not CRAN.